scripts/build-release.sh

```
@@ -138,11 +138,11 @@ if [[ -z "${LLAMA_STAGE_BUILD_DIR:-}" && -n "${SKIPPY_LLAMA_BUILD_DIR:-}" ]]; th
     export LLAMA_STAGE_BUILD_DIR="$SKIPPY_LLAMA_BUILD_DIR"
 fi
 if [[ -z "${LLAMA_STAGE_BUILD_DIR:-}" ]]; then
-    if [[ "$BACKEND" == "cpu" ]]; then
-        export LLAMA_STAGE_BUILD_DIR="$LLAMA_BUILD_ROOT/build-stage-abi-static"
-    else
-        export LLAMA_STAGE_BUILD_DIR="$LLAMA_BUILD_ROOT/build-stage-abi-$BACKEND"
-    fi
+    export LLAMA_STAGE_BUILD_DIR="$(
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)